ros_buildfarm/jenkins.py

```diff
@@ -270,8 +270,8 @@ def _diff_configs(remote_config, new_config):
     if ElementTree.tostring(remote_root) == ElementTree.tostring(new_root):
         return []
 
-    xml1 = ElementTree.tostring(remote_root, encoding='unicode')
-    xml2 = ElementTree.tostring(new_root, encoding='unicode')
+    xml1 = ElementTree.tostring(remote_root)
+    xml2 = ElementTree.tostring(new_root)
```

Comment:
this was to get it to work with python 2.7 (before I realised it was targeting python 3) so I will revert it and https://github.com/dhood/ros_buildfarm/commit/e4b4cbca1ecf40a4f2e58a8aa4f48851309f1890